doc/nrf/libraries/gazell/gzp.rst

```diff
@@ -10,9 +10,55 @@ Gazell Pairing
 The Gazell pairing library enables applications to use the Gazell Link Layer to provide a secure wireless link between Gazell nodes.
 The library is customized for pairing a device (for example, a mouse, keyboard, or remote control) with a host (typically a USB dongle) using Gazell.
 
-See the :ref:`ug_gzp` user guide for more information.
+Overview
+********
+See the :ref:`ug_gzp` user guide for more information, such as the features of this library.
 
-This library is used in the :ref:`gzp_dynamic_pairing` sample.
+This library is used in the :ref:`gzp_dynamic_pairing_host` and :ref:`gzp_dynamic_pairing_device` samples.
+
+Requirements
+************
+
+In addition to the resources required by the :ref:`ug_gzll` (see :ref:`ug_gzll_resources`), the Gazell Pairing library also employs three nRF5 peripherals:
+
+* Random Number Generator, for generating keys and tokens.
+* AES Electronic Codebook (ECB), for encryption and decryption.
+* Non-Volatile Memory Controller (NVMC), for storing of pairing parameters.
+
+In addition, Gazell Pairing employs the following Gazell Link Layer resources:
+
+* Two pipes: one for pairing and one for encrypted data transmission.
+* Gazell Pairing determines the channel set used by Gazell.
```

Comment:
@greg-fer is it because of the colon in ```Two pipes: ```  

Or now that I read it over, this should not be a bulleted list, because the ```Gazell Pairing determines...``` point should be it's own sentence?

Reply:
Combine 28 - 30 into one sentence, and 31 will be its own line (or check if it belongs elsewhere) OR it can be broken down into its own list.

Reply:
Okay, will implement this change and move 31 to the Overview section.